client/src/app/pages/dashboard/maps/map-info/map-leaderboard/map-leaderboard.component.html

```diff
@@ -19,13 +19,18 @@ <h2 class="col">LEADERBOARD</h2>
             <input nbInput fullWidth type="text" class="form-control" placeholder="Name">
           </div>
           <div class="col-md">
-            <nb-checkbox class="mr-1">W-Only</nb-checkbox>
-            <nb-checkbox class="mr-1">Backwards</nb-checkbox>
+            <!-- <nb-checkbox class="mr-1">W-Only</nb-checkbox>
+            <nb-checkbox class="mr-1">Backwards</nb-checkbox> -->
+            <nb-radio-group [(value)]="filterOption">
+              <nb-radio [value]="1">Show top 10 times</nb-radio>
+              <nb-radio [value]="2">Show ranks around me</nb-radio>
+              <nb-radio [value]="3">Show friend's time</nb-radio>
```

Comment:
These should probably be enum values, since "1" doesn't really mean much.

Something like:
```js
enum LeaderboardType {
    LEADERBOARD_TOP10 = 1,
    LEADERBOARD_AROUND = 2,
    LEADERBOARD_FRIENDS = 3
}
```